tests/ci/docker_images/linux-arm/amazonlinux-2_gcc-7x_corretto/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM amazonlinux-2:accp_base-arm
+
+SHELL ["/bin/bash", "-c"]
+
+RUN amazon-linux-extras enable corretto8
+
+# gcc 7.3.1 is the latest version versions `yum --showduplicates list gcc`
+# JRE and JDK tools are separate in Corretto 17.
+RUN set -ex && \
+    yum -y update && yum install -y \
+    gcc \
+    gcc-c++ \
+    java-1.8.0-amazon-corretto-devel \
+    java-11-amazon-corretto \
+    java-17-amazon-corretto \
+    java-17-amazon-corretto-devel && \
+    yum clean packages && \
+    yum clean metadata && \
+    yum clean all && \
+    rm -rf /tmp/* && \
+    rm -rf /var/cache/yum
+
+RUN mkdir /accp
+COPY . /accp
+WORKDIR /accp
+
+# run the gradlew script just to install gradle in the image
+RUN ./gradlew --no-daemon generateEclipseClasspath
+
+ENV CC=gcc
+ENV CXX=g++
+ENV JAVA_HOME=/usr/lib/jvm/java-11-amazon-corretto.aarch64
```

Comment:
why set the JAVA_HOME to 11 here? won't all the extending Dockerfiles take care of that?

Reply:
The ACCP build complains when `JAVA_HOME` isn't set with an error like this:
```
CMake Error at /usr/share/cmake3/Modules/FindJava.cmake:166 (message):
  Error executing java -version
Call Stack (most recent call first):
  CMakeLists.txt:12 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/ubuntu/workplace/amazon-corretto-crypto-provider/build/cmake/CMakeFiles/CMakeOutput.log".
```
It seems like CMake relies on the variable to look for the java binary.

Reply:
ok, that makes sense  that it would need to know where java lives in order to build ACCP. i suppose it doesn't matter whether we choose java 11 or 17 to build with; IIRC we can't build with java 8.